internal/worker/client/command/stream.go

```diff
@@ -0,0 +1,59 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/proto"
+	"google.golang.org/grpc"
+)
+
+type StreamCommand struct {
+	client proto.WorkerServiceClient
+	args   []string
+}
+
+func NewStreamCommand(client proto.WorkerServiceClient) Runner {
+	return &StreamCommand{
+		client: client,
+	}
+}
+
+func (c *StreamCommand) Init(args []string) {
+	c.args = args
+}
+
+func (c *StreamCommand) Run() {
+	ctx, cancel := context.WithCancel(context.Background())
+	command := proto.StreamRequest{
+		JobID: c.args[0],
+	}
+	res, err := c.client.Stream(ctx, &command, grpc.WaitForReady(true))
+	if err != nil {
+		cancel()
+		os.Stderr.WriteString(fmt.Sprintf("%v\n", err))
+		return
+	}
+	// runs the streaming in backgroud
+	go func() {
+		for {
+			out, err := res.Recv()
+			if err == nil {
+				os.Stdout.WriteString(out.Output)
+			} else {
+				os.Stderr.WriteString(fmt.Sprintf("%v\n", err))
```

Comment:
when the stream ends server-side (when the job finishes), you should get `io.EOF` or a similar error
don't output those errors